arcgis-ios-sdk-samples/Augmented reality/Navigate in AR/README.md

```diff
@@ -0,0 +1,70 @@
+# Navigate in AR
+
+Use a route displayed in the real world to navigate.
+
+![Image of Navigate in AR](navigate-ar.png)
+
+## Use case
+
+It can be hard to navigate using 2D maps in unfamiliar environments. You can use full-scale AR to show a route overlaid on the real-world for easier navigation.
+
+## How to use the sample
+
+The sample opens with a map centered on the current location. Tap the map to add an origin and a destination; the route will be shown as a line.
+
+When ready, tap the camera button to start the AR navigation. Calibrate the heading before starting to navigate.
+
+When you start, route instructions will be displayed and spoken. As you proceed through the route, new directions will be provided until you arrive.
+
+## How it works
+
+1. The map page is used to plan the route before starting the AR experience. See *Navigate route*, *Find a route* and *Offline routing* samples for a more focused demonstration of that workflow.
+2. Pass the resulting `AGSRouteResult` and the input `AGSRouteTask` and `AGSRouteParameters` to the page used for the AR navigation experience.
```

Comment:
Should that be "view" instead of "page"?

```suggestion
2. Pass the resulting `AGSRouteResult` and the input `AGSRouteTask` and `AGSRouteParameters` to the view used for the AR navigation experience.
```